app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -90,13 +90,6 @@ table.table-show {
   }
 }
 
-// Following should be set in component library.
-.form-label {
-  margin-bottom: unset;
-  font-size: 1rem;
-  font-weight: 400;
-}
```

Comment:
@justinlittman The PR these lines were in doesn't make clear to me why we needed these overrides. Can you say more?

Reply:
Per https://github.com/sul-dlss/hungry-hungry-hippo/issues/280, the form label font size and spacing needed adjustment.

Reply:
I'm not sure the mock-ups are consistent with vanilla bootstrap, though. Look at this mockup:

https://private-user-images.githubusercontent.com/131982/395730257-de7dc41e-ce01-4c68-8d60-8a0515340cf1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQzNjY3NzMsIm5iZiI6MTczNDM2NjQ3MywicGF0aCI6Ii8xMzE5ODIvMzk1NzMwMjU3LWRlN2RjNDFlLWNlMDEtNGM2OC04ZDYwLThhMDUxNTM0MGNmMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjE2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIxNlQxNjI3NTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wZDBmMWYxOWY2ZmMxZTliMWVmZDIwODlmYjc1OTA0MGFjNzM3NTg5ODc3MTE3ZTg3OTdjMTAyMTdhYmRkMmNkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.biMXT13AjwpjXe98U3-GxStg5N_JmBEJcTu-xoJG_Jw

Reply:
@justinlittman in ☝🏻, the form labels `Role term type` and `Role term` are bolded but the form labels `ORCID ID`, `First name`, and `Last name` are not bolded. Also, the amount of space between the `Role term type` label and its input is larger than the others. I guess the question is: assuming my analysis is correct here (and it may not be), are we looking to change the mock-ups to be more vanilla bootstrap, or should we make the app more faithful to the mock-ups even if that means deviating from vanilla bootstrap (i.e., some form labels styled differently than others)?

Reply:
AFAICT, `form-label` text is already `font-size: 1rem` and `font-weight: 400`. I'll look into the `margin-bottom` issue.

Reply:
I extended the `form-label` class with the `mb-0` class.